types/cross-domain-utils/utils.d.ts

```diff
@@ -0,0 +1,247 @@
+import { CrossDomainWindowType, SameDomainWindowType, DomainMatcher } from './types';
+
+export function isFileProtocol(win: SameDomainWindowType): boolean;
+
+export function isAboutProtocol(win: SameDomainWindowType): boolean;
+
+/**
+ *  Gets the parent of the specified window, if the window has a parent.
+ *
+ *  - Only returns the parent of iframes
+ *  - Returns void if the window is the top-level window
+ */
+export function getParent(win?: CrossDomainWindowType): CrossDomainWindowType | null;
```

Comment:
should it return `undefined`? (which seem to be more correct technically):
https://github.com/krakenjs/cross-domain-utils/blob/87164b88ae8d2b59cf97608af9ce68efe66c2c16/src/utils.js#L21